snmp/datadog_checks/snmp/utils.py

```diff
@@ -253,7 +253,7 @@ class OIDPrinter(object):
     """
 
     def __init__(self, oids, with_values):
-        # type: (Union[Mapping, Sequence], bool) -> None
+        # type: (Union[Mapping, Sequence[ObjectType]], bool) -> None
```

Comment:
Note for reviewers: seems mypy is not verifying this type `Sequence[ObjectType]]`, not sure why. Added this for documentation.